clients/client-python/gravitino/catalog/fileset_catalog.py

```diff
@@ -200,15 +200,14 @@ def format_fileset_request_path(namespace: Namespace) -> str:
     def to_fileset_update_request(change: FilesetChange):
         if isinstance(change, FilesetChange.RenameFileset):
             return FilesetUpdateRequest.RenameFilesetRequest(change.new_name())
-        elif isinstance(change, FilesetChange.UpdateFilesetComment):
+        if isinstance(change, FilesetChange.UpdateFilesetComment):
```

Comment:
Why do we change the `elif` to `if`? Does that mean we have to continue the remaining `if` judgment even though the first `if` is matched?

Reply:
logically it is the same because of the return, right? Let me check if it is the code style of python.

Reply:
Yeah, actually it's `no-else-return` in Pylint, it's not necessary to use `elif` in this case.

Reply:
This is suggested by pylint, the no-else-return rule
https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/no-else-return.html

Reply:
Got it.